config/packages/monolog.yaml

```diff
@@ -7,13 +7,13 @@ when@dev:
         handlers:
             main:
                 type: rotating_file
-                path: '%kernel.logs_dir%/%kernel.environment%.log'
+                path: '/var/log/mbin/%kernel.environment%.log'
```

Comment:
I noticed my dev environment broke because this is hardcoded to `/var/log/mbin`, any particular reason?

Reply:
Its because I am stupid... Shouldn't have committed that...